mri/operators/fourier/non_cartesian.py

```diff
@@ -482,8 +482,12 @@ def __init__(self, kspace_loc, shape, implementation='cpu', n_coils=1):
         """
         self.num_slices = shape[2]
         self.shape = shape
-        (kspace_plane_loc, self.z_sample_loc,
-         self.sort_pos) = get_stacks_fourier(kspace_loc)
+        (kspace_plane_loc, self.z_sample_loc, self.sort_pos, self.idx_mask_z) \
```

Comment:
why can't we make this:
```python
(kspace_plane_loc, self.z_sample_loc, self.sort_pos, self.idx_mask_z) = get_stacks_fourier(
  kspace_loc,
  self.shape,
)
```
?

Reply:
That would cross the PEP8 line length of 78 characters right?

Reply:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length
Since we are the only ones working on this project, I think it's fine to have a longer maximum line length like 99.

In the team I worked in before, because it was not an open-source project, we even used 120 characters.

Reply:
It failed the test if I change it, because of pep8. I let it like this for now, but if you insist I'll change it.